src/DataBase/EntryDatabase.cs

```diff
@@ -178,7 +181,45 @@ public async Task<IEntry> ReadAsync(Expression<Func<IEntryModel, bool>> filter,
 
         public async Task<IEntryList> ReadListAsync(int userId, EntryType entryType)
         {
-            return await ReadListAsync(x => true, userId, entryType);
+            //return await ReadListAsync(x => true, userId, entryType);
+            var list = new EntryList(entryType);
+
+            var type = entryType == 0 ? "Incomes" : "Expenses";
+
+            string query = "SELECT * FROM " + type + " WHERE UserId =" + userId;
```

Comment:
This and quite a few other variables afterwards in this method could be declared with var